packages/manager/modules/pci/src/projects/project/sidebar/sidebar.service.js

```diff
@@ -15,9 +15,9 @@ export default class Sidebar {
     return [
       ...MENU.map(item => ({
         name: item.translation,
-        title: this.$translate.instant(item.translation),
+        title: item.title,
```

Comment:
What does this have to do with the schmilblick ?

Reply:
Long story short, sidebar not handled the same way on mobile, I've started a refactor following that issue  but in the meanwhile